definitions/ext-host/definition.yml

```diff
@@ -17,6 +17,41 @@ synthesis:
       conditions:
       - attribute: eventType
         value: FlexSystemSample
+
+     # telemetry from generic ansible agent
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: AnsibleSystemSample
+
+    # ansible agent requires same guid on other generated samples
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: AnsibleNetstatSample
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: AnsibleServiceSample
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: AnsibleStorageSample 
+    - identifier: displayName
```

Comment:
without a custom dashboard for this they will use the first one defined for datadog for the summary view.
And I don't think that will match on telemetry right?

Reply:
@naxhh correct, for context in the cloud accelerator project we are using the data generated from Flex and Ansible to create entities which will then eventually be migrated to the cloud and have our core agents in the landing zone. We are using workloads to track the entities between on-prem and cloud.

Re. custom dashboard we are fine with not having any dashboard associated to the other telemetry, we just want the ensure the same guid across the event types.

Reply:
I don't have context enough to understand the deep explanation, @naxhh or @Kav91, could any of you confirm the approach reached by the PR is the desired?

Reply:
@danipl yes it should produce our desired outcome. Also as the usage is scoped to the cloud accelerator project and our build partner, it will not have any large customer impact at this stage.